docs/index.rst

```diff
@@ -1,53 +1,147 @@
+
+To help you navigate the various services offered through Ubuntu Pro, click on the tab that best represents your situation for our suggestions on which services might be of interest to you. For example, Pro includes a set of package versions that are compliant with FIPS regulations. You will only want these versions on machines that *need* to meet FIPS requirements, so you can choose to enable that stream specifically on those machines.
+
+.. tab-set::
+
+    .. tab-item:: Commercial or enterprise
+
+       * **Expanded Security Maintenance (ESM)**
+             
+         If you're using an Ubuntu LTS -- Desktop or Server -- in a commercial setting and don't want the inconvenience and downtime of upgrading to a newer LTS yet, you can get additional security support through `Expanded Security Maintenance (ESM)`_. We recommend it for everyone using Pro, and it's enabled by default. It includes two services:
+     
+         * ``esm-infra`` provides security updates for packages in the Ubuntu "main" repository (packages from Ubuntu) for five additional years beyond the standard five years of security maintenance of "main" packages.
+     
+         * ``esm-apps`` covers packages in the "universe" repository (which are from Debian and the community) for ten years after the LTS release. 
+ 
+         * Whether you need one or both of these depends on your subscription. If you're not sure which services best support the needs of your organisation, contact us for `a security assessment`_.
+
+       * **Livepatch**
+
+         Ubuntu `Livepatch`_ reduces costly unplanned maintenance by patching vulnerabilities while the system runs. It removes the need to immediately reboot the kernel after an upgrade so that you can schedule downtime for when it's convenient. This tooling is available through ``livepatch``.
+         
+         * Find out more about what Livepatch can do for your organisation in the `Livepatch documentation`_, or find out :doc:`how to enable it<howtoguides/enable_livepatch>`.
+         
+       * **Landscape** (coming soon!)
+         
+         `Landscape`_
+         
+       Want more information?
+
+    .. tab-item:: Certification and hardening
+
+       Ubuntu provides security compliance, `certifications, and hardening`_:
+       
+       * **FIPS 140-2**
+       
+         * `FIPS 140-2 Certified Modules`_ are available through ``fips``.
+         
+         * Compliant but non-certified patches are available through ``fips-updates``.
+         
+         * For more details about managing FIPS, check out this guide. <-- note we need a guide, which also includes which services are incompatible with fips   
+       
+       * **CIS benchmarking**
```

Comment:
IMHO CIS should be under USG, as USG is really what you can enable on recent LTS. For Xenial/Bionic, only CIS is available though

Reply:
Well this works until they add a new service to USG - maybe without warning us because oh well - didn't happen yet so I have nothing material against your suggestion.

Reply:
Looks good!